x/bounty/client/cli/flags.go

```diff
@@ -0,0 +1,13 @@
+package cli
+
+const (
+	FlagEncKeyFile = "enc-key"
+	FlagDesc       = "desc"
+	FlagCommissionRate = "commission_rate"
+	FlagDeposit = "deposit"
+	FlagSubmissionEndTime = "submit-end"
+	FlagJudgingEndTime = "judge-end"
+	FlagClaimEndTime = "claim-end"
+
+	dateLayout = "1999-12-31"
```

Comment:
Seems "1999-12-31" doesn't work,  while "2006-01-02" works.

Reply:
hmm could you debug it?

Reply:
Seems it's golang time parsing issue. I tested separately with a small piece of code in https://go.dev/play/
```func main() {
	const layout = "2006-01-08"
	tm, _ := time.Parse(layout, "2014-02-04")
	fmt.Println(tm)
}
"2006-01-08" doesn't work well, while "2006-01-07" works.